js/libs/core.js

```diff
@@ -883,6 +888,63 @@ Fliplet.FormBuilder = (function() {
         component.methods.openFilePicker = component.methods._openFilePicker;
       }
 
+      component.methods._getFieldOptions = function() {
+        var $vm = this;
```

Comment:
@armine-fliplet, again, it's not needed as you never use it elsewhere, where context could be lost